Makefile

```diff
@@ -375,13 +375,15 @@ black: requirements .black-check
 		echo "Running black on" $$component; \
 		echo "==========================================================="; \
 		. $(VIRTUALENV_DIR)/bin/activate ; black --check --config pyproject.toml $$component/ || exit 1; \
+		. $(VIRTUALENV_DIR)/bin/activate ; black $$(grep -rl '^#!/.*python' $$component/bin) || exit 1; \
```

Comment:
This is one out of 4 places where grep fails due to the non-existing bin-directory. Example: `grep: contrib/runners/announcement_runner/bin: No such file or directory`

Reply:
Same here (see comment below).

If you feel strongly about it, I can add if back (but do keep in mind that this makes the Makefile a bit more messy :)).